ui/lib/apps/SlowQuery/pages/Detail/DetailTabCopr.tsx

```diff
@@ -30,9 +21,22 @@ export default function TabCopr({ data }: ITabCoprProps) {
       key: 'cop_wait_addr',
       value: data.cop_wait_addr,
     },
+    genShortValueTooltipValueItem(data, 'rocksdb_block_cache_hit_count'),
+    genShortValueTooltipValueItem(data, 'rocksdb_block_read_byte'),
+    genShortValueTooltipValueItem(data, 'rocksdb_block_read_count'),
+    genShortValueTooltipValueItem(data, 'rocksdb_delete_skipped_count'),
+    genShortValueTooltipValueItem(data, 'rocksdb_key_skipped_count'),
   ]
   const columns = valueColumns('slow_query.fields.')
   return (
     <CardTable cardNoMargin columns={columns} items={items} extendLastColumn />
   )
 }
+
+// TODO: refactor items code gen for all DetailsList
+function genShortValueTooltipValueItem(data: SlowqueryModel, key: string) {
```

Comment:
What if the field does not exist? It would be confusing to display the value as 0, instead of not display at all.